start.js

```diff
@@ -13,6 +13,7 @@ const listenAddressDocker = '0.0.0.0'
 const watch = require('./lib/watch')
 const parseArgs = require('./args')
 const { exit, requireFastifyForModule, requireServerPluginFromPath, showHelpForCommand } = require('./util')
+const inspector = require('inspector')
```

Comment:
We are still supporting node 6, and this is not available there. Would you mind loading this module only in that case?

Reply:
Sure, I undestand it is a experimental feature of nodejs but it was the simplest way to do that :)

Reply:
That's brilliant, definitely +1!